Cython/Compiler/TreePath.py

```diff
@@ -11,10 +11,7 @@
 import operator
 import sys
 
-if sys.version_info[0] >= 3:
-    _unicode = str
-else:
-    _unicode = unicode
+_unicode = str
```

Comment:
This is worth removing (and "inlining") as well.

Reply:
Could you (suggest how to) implement that?

Reply:
I think just a find/replace in your favourite text editor for `_unicode` -> `str`

Reply:
So it says `str = str`?

Reply:
It was even only one occurrence (see 9fff260), here's the full diff:

```diff
diff --git a/Cython/Compiler/TreePath.py b/Cython/Compiler/TreePath.py
index b1371c249a5..007581890d9 100644
--- a/Cython/Compiler/TreePath.py
+++ b/Cython/Compiler/TreePath.py
@@ -11,8 +11,6 @@
 import operator
 import sys
 
-_unicode = str
-
 path_tokenizer = re.compile(
     r"("
     r"'[^']*'|\"[^\"]*\"|"
@@ -165,7 +163,7 @@ def select(result):
                     continue
                 if attr_value == value:
                     yield attr_value
-                elif (isinstance(attr_value, bytes) and isinstance(value, _unicode) and
+                elif (isinstance(attr_value, bytes) and isinstance(value, str) and
                         attr_value == value.encode()):
                     # allow a bytes-to-string comparison too
                     yield attr_value
```